docs-2.0/8.service-tuning/compaction.md

```diff
@@ -0,0 +1,63 @@
+# Compaction
+
+This document gives some information to compaction.
+
+## Introduction to compaction
+
+In Nebula Graph, compaction is the most important background process. Compaction has an important effect on performance.
+
+Compaction re-reads the data that is written on the hard disk, then re-organizes the data structure and the indexes to make the data more easier to read. The read performance can increase by times after compaction. Thus, to get high read performance, trigger compaction manually when writing a large amount of data into Nebula Graph. Note that compaction leads to long time hard disk IO, we suggest that you do compaction during business troughs (for example, early morning).
+
+Nebula Graph has these two type of compaction:
+
+- Automatic compaction
+    Automatic compaction is done when the system reads, writes data, or the system restarts. The automatic compaction is enabled by default. But it can be enabled during business peaks, causing unexpected IO occupancy that has an unwanted effect on the performance. To get full manual control of compaction, use this statement:
```

Comment:
> Automatic compaction is done when the system reads, writes data, or the system restarts.

Automatic compaction is done when the system reads data, writes data, or restarts.

> business peaks

Usually, we use "peak hours". The other side of the coin is "the off-peak hours".

> But it can be enabled during business peaks, causing unexpected IO occupancy that has an unwanted effect on the performance.

But once triggered during peak hours, it can decrease the service performance (dramatically? I'm not sure).